runtime/gc_vlhgc/ProjectedSurvivalCollectionSetDelegate.cpp

```diff
@@ -600,6 +606,8 @@ void
 MM_ProjectedSurvivalCollectionSetDelegate::rateOfReturnCalculationAfterSweep(MM_EnvironmentVLHGC *env)
```

Comment:
I suspect that this math for arrays is broken, since array reserved regions are freed at GMP Root Clearing phase, hence before Sweep. I'm guessing that Before and After math will return the same number.

But let's not worry about that in this PR - just make the calculation (relatively) simple and correctly account for shared/fractional array reserved regions.